mkdocs.yml

```diff
@@ -3,7 +3,9 @@ site_description: A project template and directory structure for Python data sci
 site_url: https://mila-iqia.github.io/ResearchTemplate/
 repo_url: https://www.github.com/mila-iqia/ResearchTemplate
 # edit_uri: edit/master/docs
-
+nav:
+  - Home: index.md
+  - Profiling your code: docs/examples/profiling.ipynb
```

Comment:
Shouldn't this be added in the `index.md` file? I imagine that this would add the profiling notebook at the top level of the docs.

Reply:
Edit: SUMMARY.md (which is the navigation bar), not `index.md`

Reply:
fixed in 7d5c1d2e1c437ed76f58c76a78804bbc8c1cdb8c